### PR TITLE
Update url links in rig/rag notebooks

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: notebooks/data_gemma_rag.ipynb [hunk 1]
```diff
@@ -7300,7 +7300,7 @@
         "DC_API_KEY = userdata.get('DC_API_KEY')\n",
         "dc = dg.DataCommons(api_key=DC_API_KEY)\n",
         "\n",
-        "# Get Gemini 1.5 Flash mode\n",
+        "# Get Gemini 1.5 Flash model\n",
```
Comment (chejennifer) [inline notebooks/data_gemma_rag.ipynb]: curious, why flash and not pro?

**Review by chejennifer — APPROVED** [review]
nice, jsut one question about flash vs pro

File: notebooks/data_gemma_rag.ipynb (hunk 1 above)
Comment (shifucun) [reply notebooks/data_gemma_rag.ipynb]: updated to pro

**Comment by shifucun** [comment]
thanks for review!

**Pull request merged**